noaa_semantic_search/test_procedures/hybrid_search.json

```diff
@@ -57,4 +57,14 @@
       "schedule": [
         {{ benchmark.collect(parts="semantic-search-common/search-with-sort.json") }}
       ]
+    },
+    {
+      "name": "hybrid-search-create-index-and-complex-queries",
+      "description": "Run hybrid search queries with multiple sub-queries, aggregations and sort.",
+      "default": false,
+      "schedule": [
+        {{ benchmark.collect(parts="semantic-search-common/create-index-ingest-docs.json") }},
```

Comment:
A suggestion: lets not add ingestion procedures with this method. Just complex query is enough. 

If we need to do ingestion we just provide create-and-index procedure. This way it gives clear numbers of search.

Reply:
I want this procedure to be a default one stop for nightly runs, so it needs to create resources